docs/resources/client/models/bakedmodel.md

```diff
@@ -43,6 +43,8 @@ Other methods in `BakedModel` that you may override and/or query include:
 | `ItemOverrides getOverrides()`                                                | Returns the [`ItemOverrides`][itemoverrides] associated with this model. This is only relevant on item models.                                                                                                                                                                                                                                                                                              |
 | `ModelData getModelData(BlockAndTintGetter, BlockPos, BlockState, ModelData)` | Returns the model data to use for the model. This method is passed an existing `ModelData` that is either the result of `BlockEntity#getModelData()` if the block has an associated block entity, or `ModelData.EMPTY` if that is not the case. This method can be used for blocks that need model data, but do not have a block entity, for example for blocks with connected textures.                    |
 | `TextureAtlasSprite getParticleIcon(ModelData)`                               | Returns the particle sprite to use for the model. May use the model data to use different particle sprites for different model data values. NeoForge-added, replacing the vanilla `getParticleIcon()` overload with no parameters.                                                                                                                                                                          |
+| `ChunkRenderTypeSet getRenderTypes(BlockState, RandomSource, ModelData)`      | Returns a `ChunkRenderTypeSet` containing the render type(s) to use for rendering the block model. By default falls back to the normal model-bound render type lookup, which always yields a set with one element. Note that `ChunkRenderTypeSet` is not actually a set, but an ordered `Iterable<List>` (so more of a list). Only used for block models, item models use the overload below.               |
```

Comment:
> By default falls back to the normal model-bound render type lookup

What is "normal model-bound render type lookup" referring to? This should clarify that certain vanilla models are patched to use the render type specified in the model JSON if present and otherwise fall back to an `ItemBlockRenderTypes` lookup with the given `BlockState`. This also applies to the item overload below, except that the lookup is done with the given `ItemStack`.

> Note that `ChunkRenderTypeSet` is not actually a set, but an ordered `Iterable<List>`

Apart from implementation details being completely irrelevant here, the `ChunkRenderTypeSet` is actually a set in that it is backed by a `BitSet` and can only contain each `RenderType` exactly once.